app/views/posts/_editor.haml

```diff
@@ -12,7 +12,7 @@
     #current-icon-dropdown= dropdown_icons(item, character_galleries)
     .post-info-text
       .post-character{class: ('hidden' unless item.character), data: { character_id: item.character_id || '', alias_id: item.character_alias_id || '' }}
-        %span#name= item.name
+        %span#name= link_to item.character_name, character_path(item.character_id)
```

Comment:
You do in fact need to use `item.name` here, it handles aliases gracefully (it's defined here: https://github.com/Marri/glowfic/blob/master/app/concerns/writable.rb#L47)